docs/ci.md

```diff
@@ -1,25 +0,0 @@
-# Continuous Integration
```

Comment:
Besides making reference to the deleted env vars, this file is very out-of-date and is no longer relevant.